src/views/credentials/viewCredentialsList.tsx

```diff
@@ -349,6 +371,17 @@ const CredentialsListView: React.FunctionComponent = () => {
           name: pendingDeleteCredential?.name
         })}
       </Modal>
+      <AddCredentialModal
+        isOpen={credentialBeingEdited !== undefined}
+        credential={credentialBeingEdited}
+        onClose={() => setCredentialBeingEdited(undefined)}
+        onSubmit={(payload: CredentialType) =>
+          editCredentials(payload).finally(() => {
+            queryClient.invalidateQueries({ queryKey: [API_CREDS_LIST_QUERY] });
+            setCredentialBeingEdited(undefined);
```

Comment:
required: the new form blocking toast notification PR merged #517 . We need to rebase this PR, and this `finally` needs to be updated similar to

https://github.com/quipucords/quipucords-ui/blob/main/src/views/credentials/viewCredentialsList.tsx#L357